### ISSUE TITLE
Write policy for starting/stopping/restarting dbus-org.fedoraproject.FirewallD1.service

### ISSUE THREAD

**Issue by ykhodorkovskiy** [TresysTechnology/clip#196]
```
type=AVC msg=audit(1438549165.196:1961): avc:  denied  { read write } for  pid=14300 comm="iptables" path=2F746D702F666669685262773059202864656C6574656429 dev="dm-4" ino=20 scontext=system_u:system_r:iptables_t:s0 tcontext=system_u:object_r:firewalld_tmp_t:s0 tclass=file
type=AVC msg=audit(1438549165.196:1961): avc:  denied  { rlimitinh } for  pid=14300 comm="iptables" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:iptables_t:s0 tclass=process
type=AVC msg=audit(1438549165.196:1961): avc:  denied  { siginh } for  pid=14300 comm="iptables" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:iptables_t:s0 tclass=process
type=AVC msg=audit(1438549165.196:1961): avc:  denied  { noatsecure } for  pid=14300 comm="iptables" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:iptables_t:s0 tclass=process
type=AVC msg=audit(1438549165.358:2018): avc:  denied  { read write } for  pid=14356 comm="rmmod" path=2F746D702F666669685262773059202864656C6574656429 dev="dm-4" ino=20 scontext=system_u:system_r:insmod_t:s0 tcontext=system_u:object_r:firewalld_tmp_t:s0 tclass=file
type=AVC msg=audit(1438549165.358:2018): avc:  denied  { rlimitinh } for  pid=14356 comm="rmmod" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549165.358:2018): avc:  denied  { siginh } for  pid=14356 comm="rmmod" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549165.358:2018): avc:  denied  { noatsecure } for  pid=14356 comm="rmmod" scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549166.585:2020): avc:  denied  { rlimitinh } for  pid=14385 comm="systemd-cgroups" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:systemd_cgroups_t:s0 tclass=process
type=AVC msg=audit(1438549166.585:2021): avc:  denied  { siginh } for  pid=14385 comm="systemd-cgroups" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:systemd_cgroups_t:s0 tclass=process
type=AVC msg=audit(1438549166.585:2022): avc:  denied  { noatsecure } for  pid=14385 comm="systemd-cgroups" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:systemd_cgroups_t:s0 tclass=process
type=AVC msg=audit(1438549186.259:2023): avc:  denied  { rlimitinh } for  pid=14388 comm="firewalld" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:firewalld_t:s0 tclass=process
type=AVC msg=audit(1438549186.259:2023): avc:  denied  { siginh } for  pid=14388 comm="firewalld" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:firewalld_t:s0 tclass=process
type=AVC msg=audit(1438549186.259:2023): avc:  denied  { noatsecure } for  pid=14388 comm="firewalld" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:firewalld_t:s0 tclass=process
type=AVC msg=audit(1438549186.296:2024): avc:  denied  { getattr } for  pid=14388 comm="firewalld" name="/" dev="selinuxfs" ino=1 scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:object_r:security_t:s0 tclass=filesystem
type=AVC msg=audit(1438549186.297:2025): avc:  denied  { getattr } for  pid=14388 comm="firewalld" path="/sys/fs/selinux" dev="selinuxfs" ino=1 scontext=system_u:system_r:firewalld_t:s0 tcontext=system_u:object_r:security_t:s0 tclass=dir
type=AVC msg=audit(1438549186.349:2026): avc:  denied  { rlimitinh } for  pid=14390 comm="modprobe" scontext=system_u:system_r:iptables_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549186.349:2026): avc:  denied  { siginh } for  pid=14390 comm="modprobe" scontext=system_u:system_r:iptables_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549186.349:2026): avc:  denied  { noatsecure } for  pid=14390 comm="modprobe" scontext=system_u:system_r:iptables_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549186.351:2027): avc:  denied  { search } for  pid=14390 comm="modprobe" scontext=system_u:system_r:insmod_t:s0 tcontext=system_u:system_r:kernel_t:s0 tclass=key
type=AVC msg=audit(1438549186.354:2028): avc:  denied  { rlimitinh } for  pid=14393 comm="modprobe" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549186.354:2029): avc:  denied  { siginh } for  pid=14393 comm="modprobe" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438549186.355:2030): avc:  denied  { noatsecure } for  pid=14393 comm="modprobe" scontext=system_u:system_r:kernel_t:s0 tcontext=system_u:system_r:insmod_t:s0 tclass=process
type=AVC msg=audit(1438546648.800:1619): avc:  denied  { read write } for  pid=13723 comm="rmmod" path=2F746D702F666669397672616741202864656C6574656429 dev="dm-4" ino=20 scontext=system_u:system_r:insmod_t:s0 tcontext=system_u:object_r:firewalld_tmp_t:s0 tclass=file
type=AVC msg=audit(1438546648.862:1620): avc:  denied  { read write } for  pid=13725 comm="rmmod" path=2F746D702F666669397672616741202864656C6574656429 dev="dm-4" ino=20 scontext=system_u:system_r:insmod_t:s0 tcontext=system_u:object_r:firewalld_tmp_t:s0 tclass=file
```

**Comment by tomhurd** [comment]
Is firewalld a service that will be deployed on clip? Why not just use
iptables? Is firewalld required by something else?

**Comment by ykhodorkovskiy** [comment]
@tomhurd firewalld is a front end for iptables in RHEL 7 iirc.

**Comment by tomhurd** [comment]
Yes, but why not use iptables by itself?

**Comment by ykhodorkovskiy** [comment]
Then we are not really keeping in line with being CLIP for RHEL 7. I would expect someone that is using CLIP to expect similar functionality to what is available in RHEL 7.

**Comment by tomhurd** [comment]
Does firewalld support secmark?

I don't see anything on their wiki: http://fedoraproject.org/wiki/FirewallD

**Comment by ykhodorkovskiy** [comment]
Since it's a frontend  for iptables, you can use secmark by enabling the iptables service which will use the iptables static rules.

http://fedoraproject.org/wiki/FirewallD#Using_static_firewall_rules_with_the_iptables_and_ip6tables_services

**Comment by mpalmi** [comment]
I'm able to stop this on https://github.com/mpalmi/clip/commit/64322f04c002c00f5e6f71e09886de8565b5e632, but when I go to start it, I get a bunch of `Request for unknown module key 'Red Hat Enterprise Linux kernel signing key 00aa5f56c587bd82f2f99d64a73dd1e9e0d334a' err -13` errors.

**Comment by ykhodorkovskiy** [comment]
This was fixed in #191
